jobs/build/refresh-images/Jenkinsfile

```diff
@@ -112,14 +135,76 @@ images:update-dockerfile
   --push
   """
 
-                buildlib.oit """
+//
+// start retry region
+//
+                BUILD_CONTINUED = false
+                waitUntil {
+                    try {
+                        exclude = ""
+                        if (BUILD_EXCLUSIONS != "") {
```

Comment:
BUILD_EXCLUSIONS would needs to be a Jenkins parameter.

Reply:
Added to the parameter section.  Defaults to "".  We would need a separate PR to update callers for the job to provide other values.